src/components/wallet-card.tsx

```diff
@@ -83,64 +68,67 @@ export const WalletCard = ({address}: BalanceProps) => {
       colorTo={wallet?.colorTo}
       colorPattern={wallet?.colorPattern}
       pattern={wallet?.pattern}
-      style={page.container}
-      width={Dimensions.get('window').width - 40}
+      style={styles.container}
+      width={screenWidth - 40}
       onLoad={() => {
         setCardState('laded');
       }}>
-      <View style={[page.topNav, !wallet.mnemonicSaved && page.marginBottom]}>
-        <Text t12 style={page.name} ellipsizeMode="tail" numberOfLines={1}>
+      <View
+        style={[styles.topNav, !wallet.mnemonicSaved && styles.marginBottom]}>
+        <Text t12 style={styles.name} ellipsizeMode="tail" numberOfLines={1}>
           {wallet.name || 'name'}
         </Text>
-        <IconButton onPress={onPressQR} style={page.qrButton}>
-          <QRCode color={LIGHT_GRAPHIC_BASE_3} />
+        <IconButton onPress={onPressQR} style={styles.qrButton}>
+          <Icon name="qr_code" color={Color.graphicBase3} />
         </IconButton>
-        <CopyButton style={page.copyButton} value={wallet.address}>
-          <Text t14 style={page.address}>
+        <CopyButton style={styles.copyButton} value={wallet.address}>
+          <Text t14 color={Color.textBase3}>
             {formattedAddress}
           </Text>
-          <Copy color={LIGHT_GRAPHIC_BASE_3} style={page.marginLeft} />
+          <Icon
+            name="copy"
+            color={Color.graphicBase3}
+            style={styles.marginLeft}
+          />
```

Comment:
Determine the size of the icon (`i<N>` prop)

Reply:
done